Adding font files and minimal CSS

kimberlyjk opened the pull request:
- added a style.css file with a couple new fonts connected. 
- added script to the index.html to connect style.css file
- added 22 font files

kimberlyjk committed:
able to get query param for call '/overview'

kimberlyjk committed:
Merge branch 'APIcalls' of https://github.com/ArrrrWeBeAnneBonny/overview into APIcalls

hannahmanfredi commented on client/overview.jsx:
```diff
@@ -1,15 +1,29 @@
 class Overview extends React.Component {
```
Ran your code! Getting one error on client out of the gate = 'Cannot GET /favicon.ico:1'. Are you trying to load a favicon?

kimberlyjk replied:
I don't think so because i dont know what that is... I will keep a look out for it though.

hannahmanfredi commented on public/index.html:
```diff
@@ -1,15 +1,11 @@
   <div id='overview'></div>
 </body>
 <script type="module" src='../dist/overview.js'></script>
 <script crossorigin src="https://unpkg.com/react@17/umd/react.development.js"></script>
```
Your code is running fine and I'm not sure if this is a real convention or not but I generally but scripts requiring cdn's inside the <head> rather than below the <body>

kimberlyjk replied:
inside, which? I just put them on the page where I've seen others?

hannahmanfredi commented on server/index.js:
```diff
@@ -54,7 +54,7 @@ app.patch('/overview/:campId/owner', async (req, res) => {
 });
 
-app.patch('/overview/:campId/pricing', async (req, res) => {
+app.get('/overview/:campId/pricing', async (req, res) => {
```
I noticed you were using PATCH on a previous . Curious what made you switch to GET?!

kimberlyjk replied:
I'm not actually updating anything so It's just a GET.

kimberlyjk committed:
Only public folder files

Previously there was code from server and client side in this commit. trying to pull apart the fonts in it.

hannahmanfredi reviewed (commented):
Made a couple of comments here. Seems like your css is loading correctly and aligned with Hipcamp's overview section and use of font-family:"Calibre"!
